modules/ui/app/scripts/services/UtilitiesFactory.js

```diff
@@ -1151,8 +1155,14 @@ angular.module('wasabi.services').factory('UtilitiesFactory', ['Session', '$stat
                     }
                 }
                 return nameList;
+            },
+
+            displaySuccessWithCacheWarning: function(title, extraMsg) {
+                var msg = extraMsg + '  PLEASE NOTE that this change may not be available for batch assignment calls for up to 5 minutes.';
```

Comment:
"batch assignment" term might be confusing to customers. @calvinwma please advise on a more customer-friendly term